lib/identity_config.rb

```diff
@@ -213,7 +218,7 @@ def self.build_store(config_map)
     config.add(:lexisnexis_trueid_timeout, type: :float)
     config.add(:lexisnexis_threatmetrix_api_key, type: :string)
     config.add(:lexisnexis_threatmetrix_base_url, type: :string)
-    config.add(:lexisnexis_threatmetrix_enabled, type: :string)
+    config.add(:lexisnexis_threatmetrix_enabled, type: :boolean)
```

Comment:
caught another one that would have bitten us! this check is already worth the time to make it